pkg/kapp/cmd/core/deps_factory.go

```diff
@@ -39,36 +38,21 @@ func NewDepsFactoryImpl(configFactory ConfigFactory, ui ui.UI) *DepsFactoryImpl
 		printTargetOnce: &sync.Once{}}
 }
 
-func (f *DepsFactoryImpl) DynamicClient() (dynamic.Interface, error) {
+func (f *DepsFactoryImpl) DynamicClient(warnings bool) (dynamic.Interface, error) {
```

Comment:
Do we wanna have something like `DynamicClientOpts` which we pass to this function  or is a parameter fine because we are not looking to add more options right now?